### PR TITLE
[internal-update] add-core-release-notes-stubs-16-17

### PR DESCRIPTION
**Pull request by dfitzmau**
Added Release Note document stubs to the `main` branch for 4.17 and 4.16.

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ocp-4-16-release-notes"]
+= {product-title} {product-version} release notes
+include::_attributes/common-attributes.adoc[]
+:context: release-notes
+
+Do not add or edit release notes here. Edit release notes directly in the branch
+that they are relevant for.
+
+Release note changes should be added/edited in their own PR.
```
Comment (kowen-rh) [inline release_notes/ocp-4-16-release-notes.adoc:10]: I'd use "added or edited" for consistency.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+This file is here to allow builds to work.
```
Comment (kowen-rh) [inline release_notes/ocp-4-16-release-notes.adoc:12]: This statement feels unnecessary to me, but feel free to disagree. :slightly_smiling_face:

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ocp-4-16-release-notes"]
+= {product-title} {product-version} release notes
+include::_attributes/common-attributes.adoc[]
+:context: release-notes
+
+Do not add or edit release notes here. Edit release notes directly in the branch
+that they are relevant for.
```
Comment (kowen-rh) [inline release_notes/ocp-4-16-release-notes.adoc:8]: Is there a reason that this is hard wrapped?

**Review by kowen-rh — COMMENTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:10]: :heavy_check_mark:

File: release_notes/ocp-4-16-release-notes.adoc (hunk 2 above)
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:12]: Ill leave it in the file is missed which can easily happen when operating on an enterprise branch and forgetting to update `main`.

File: release_notes/ocp-4-16-release-notes.adoc (hunk 3 above)
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc]: :heavy_check_mark:

**Review by skopacz1 — APPROVED** [review]
LGTM

**Pull request merged**